cchecker_web/api.py

```diff
@@ -32,6 +37,76 @@ def get_tests():
     '''
     Returns the listing of tests for compliance checker
     '''
+    tests = populate_tests()
+    return json.dumps(tests), 200, {"Content-Type": "application/json"}
+
+
+@api.route('/api/run')
+def execute_job():
+    '''
+    'Rest' endpoint for running compliance checker. Only accepts DAP urls.
+    '''
+    # Get parameters
+    report_format = request.args.get('report_format', 'json')
+    test = request.args.get('test', None)
+    url = request.args.get('url', None)
+
+    # Check report formats
+    accepted_formats = ['json', 'html']
+    if report_format not in accepted_formats:
+        err_msg = ("Report format '{0}' not available. "
+                   "Please choose from {1}".format(report_format, accepted_formats))
+        return jsonify({'Error': err_msg}), 400
```

Comment:
is this the same style of error reporting I do in the other methods? I typically keep error lowercase.